entrypoint.sh

```diff
@@ -45,11 +45,12 @@ ln -sf /dev/stdout /var/log/nginx/access.log \
   && ln -sf /dev/stderr /var/log/nginx/error.log
 
 # Set linked mysql container as mysql host
-echo -e "[client]\nhost=mysql" > ~/.my.cnf
+
+echo -e "[client]\nhost=$MYSQL_HOST" > ~/.my.cnf
 
  # Wait for the mysql container to be ready
-while ! nc -z mysql 3306; do
-  echo "Waiting for mysql to start";
+while ! mysqlshow -u$MYSQL_USER -p$MYSQL_PASSWORD > /dev/null 2>&1; do
```

Comment:
Why do you want to change this? Is it because we're not sure netcat is installed?